lib/lrmd/lrmd_client.c

```diff
@@ -1981,11 +1981,44 @@ lrmd_api_get_metadata(lrmd_t * lrmd,
     return generic_get_metadata(class, provider, type, output);
 }
 
+static const char*
+lrmd_api_get_version(lrmd_t * lrmd,
+                     const char *class,
+                     const char *provider,
+                     const char *type, enum lrmd_call_options options)
+{
+    const char *version = NULL;
+    char *output = NULL;
+
+    lrmd_api_get_metadata(lrmd, class, provider, type, &output, options);
```

Comment:
This should be called via the lrmd object rather than directly, for future-proofing, in case there are ever differing implementations of get_metadata().